ftplugin/pandoc.vim

```diff
@@ -8,6 +8,8 @@ if exists("b:pandoc_loaded") && b:pandoc_loaded == 1
     finish
 endif
 
+compiler pandoc
```

Comment:
This should probably exist in a `compiler` module (so, in `autoload/pandoc/compiler.vim`), the contents of which should be

```
function! pandoc#compiler#Init()
     compiler pandoc
endfunction
```